src/journey-maps/angular/services/map/map-transfer-service.ts

```diff
@@ -1,43 +1,36 @@
 import { Injectable } from '@angular/core';
 import { FeatureCollection } from 'geojson';
-import { GeoJSONSource, Map as MaplibreMap } from 'maplibre-gl';
+import { Map as MaplibreMap } from 'maplibre-gl';
 
-import { SBB_WALK_SOURCE } from '../constants';
+import { SbbRouteSourceService } from '../source/route-source-service';
+import { SbbTransferSourceService } from '../source/transfer-source-service';
+import { isV1Style } from '../source/util/style-version-lookup';
 
 import { SBB_EMPTY_FEATURE_COLLECTION } from './map-service';
 
 @Injectable({ providedIn: 'root' })
 export class SbbMapTransferService {
-  private _data: FeatureCollection;
+  constructor(
+    private _transferSourceService: SbbTransferSourceService,
+    private _routeSourceService: SbbRouteSourceService
+  ) {}
 
   updateTransfer(
     map: MaplibreMap,
     featureCollection: FeatureCollection = SBB_EMPTY_FEATURE_COLLECTION
   ): void {
-    this._getSource(map).setData(featureCollection);
-    this._data = featureCollection;
+    if (isV1Style(map)) {
+      this._transferSourceService.updateTransferV1(map, featureCollection);
+    } else {
+      this._routeSourceService.updateTransferV2(map, featureCollection);
+    }
   }
 
-  // If we enter the station on another floor than '0' then the outdoor route should be displayed
```

Comment:
Is this no longer the case? Just removing documentation should be carefully considered.

Reply:
the logic has moved to 2 other classes/methods. The same comment is now in both of those places.
- SbbTransferSourceService.updateOutdoorWalkFloorV1
- SbbRouteSourceService.updateOutdoorWalkFloorV2